shell/app/modules/msp/monitor/trace-insight/pages/trace-querier/trace-querier.tsx

```diff
@@ -163,11 +167,13 @@ const TraceInsightQuerier = () => {
   const renderUrlEditor = () => {
     const selectBefore = (
       <FormItem
+        className="mb-0 -mt-0.5 h-7"
```

Comment:
`-mt` is it correct?

Reply:
![image](https://user-images.githubusercontent.com/3955437/129138195-9b47260e-c081-4c85-95ba-7d267cfd2081.png)